modules/cnv-understanding-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-logs-events.adoc
+
+[id="cnv-understanding-logs_{context}"]
+= Understanding logs
+
+Logs are collected for {product-title} builds, deployments, and pods. 
```

Comment:
Are these logs any different from the OCP logs?

Reply:
Don't believe so.